src/locale/i18nProvider.web.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import {I18nProvider as DefaultI18nProvider} from '@lingui/react'
+import {i18n} from '@lingui/core'
+import {useLocaleLanguage} from './i18n.web'
+
```

Comment:
This file doesn't need to be forked right? The bundler should already pick up the right version of `./i18n` from the original one.

Reply:
It was throwing errors for me in native because when the bundler was compiling the `async import` it would throw an error

Reply:
Actually sorry -- you're right. I got confused.